lib/uplink/uplink.go

```diff
@@ -158,8 +165,12 @@ func (u *Uplink) OpenProject(ctx context.Context, satelliteAddr string, encrypti
 		return nil, Error.New("failed to create redundancy strategy: %v", err)
 	}
 	segments := segments.NewSegmentStore(metainfo, nil, rs, maxBucketMetaSize.Int(), maxBucketMetaSize.Int64())
+	var encryptionKey storj.Key
+	if opts != nil {
+		encryptionKey = opts.Volatile.EncryptionKey
```

Comment:
I'm not sure if this case would arise but I feel like there should be a check that `opts.Volatile` isn't nil so that there won't be panics.

Reply:
oh i see what you mean, yeah! opts.Volatile is actually a struct, so i don't think it's possible to even compare it to nil but it might be worth a comment?

Reply:
oh I see- I think it should be fine then.